style.css

```diff
@@ -609,3 +612,93 @@ hr {
   width: 340px;
   height: 0.2px;
 }
+
+.contact-me {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 70px;
+  padding-right: 10px;
+  border-top-left-radius: 60px;
+  background-color: #6070ff;
+  width: 330px;
+  margin-top: 200px;
+
+  /* margin-left: 10px; */
+  background-image: url("./Image/contact\ form\ shapes-mobile@2x.png");
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: 230px 730px;
+  background-position-x: 119px;
+  background-position-y: 20px;
+}
+
+.contact-title {
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  color: #fff;
+  text-align: center;
+}
+
+.contact-text {
+  color: #ebebff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  text-align: center;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: -27px;
+  margin-bottom: 20px;
+}
+
+.form-container {
+  margin-left: 25px;
+}
+
+.first-name {
+  color: #172b4d;
+  padding: 10px 16px;
+  width: 260px;
+  border-radius: 8px;
+  border-width: 0;
+  margin-top: 20px;
+}
+
+.contact-textarea {
+  color: #b3bac5;
+  margin-top: 18px;
+  border-radius: 8px;
+  border-width: 0;
+  padding: 10px 16px;
+  resize: none;
+}
+
+.git-in-touch {
+  color: #6070ff;
+  margin-top: 20px;
+  margin-bottom: 45px;
+  border-radius: 8px;
+  border-width: 0;
+  padding: 12px 16px;
+}
+
```

Comment:
- [ ] Please notice we have to stick to the original design given in figma as much as possible, currently your get in touch button is not matching the original design, please fix the `font-family`, `font-size` and `font-weight` of the button to match the original design, it should be 17px, 500 and Poppins respectively